Makefile

```diff
@@ -190,7 +190,15 @@ payload: bots
 
 .PHONY: images
 images: bots
-	bots/image-download $(TEST_OS) debian-stable ubuntu-stable fedora-41
+	# Download cloud images
+	bots/image-download debian-stable ubuntu-stable fedora-41
```

Comment:
Unrelated to this PR, but is that hardcoding of f-41 necessary? That'll just bitrot over time, no? And if you test rawhide or a compose, do you really need it?

Reply:
We are using fedora-41 image, debian-stable and ubuntu-stable images as a base images for the target disk of the installation , and then rawhide-iso for the installer media. 
So the user might want Fedora rawhide, after shrinking their pre-existing Fedora/Ubuntu/Debian installation.

I guess I can replace existing test to just use the Fedora rawhide image instead of the Fedora-*. It's just weird for the user to want to install Fedora-rawhide next to existing Fedora-rawhide (i think it might not even be supported by Anaconda currently)

Reply:
It's fine, and I figure you'll learn when to move it soon enough once we'll drop fedora-41 (and forgot about this reference, as it doesn't appear in bots testmap). :grin: